Formula/ipython@5.rb

```diff
@@ -99,7 +145,16 @@ class IpythonAT5 < Formula
   end
 
   def install
-    virtualenv_install_with_resources
+    venv = virtualenv_create(libexec)
+
+    %w[appnope backports.shutil_get_terminal_size decorator enum34 ipython_genutils pathlib2 pexpect pickleshare prompt_toolkit ptyprocess Pygments scandir simplegeneric six traitlets wcwidth].each do |r|
```

Comment:
How do we know what to put on this list vs. what to put on the other list below?